meta-mender-commercial/recipes-mender/mender-gateway/mender-gateway.inc

```diff
@@ -0,0 +1,49 @@
+LICENSE = "CLOSED"
```

Comment:
While this works, it is technically wrong. `LICENSE="CLOSED"` means to Yocto: "don't care about this". It is a common misconception that it means "closed/commerial source". This probably does not mandate a real change, but in the future we should add and ship the real license that this is under.

Reply:
Interesting, we didn't know about this. I created [MEN-5517](https://tracker.mender.io/browse/MEN-5517) to follow-up.

Reply:
Interesting, I did not know that. In any case, we need to direct such improvements to master. This PR is more about making sure the cherry-picks from master into dunfell are sensible. If we fix such things on master, we will also fix them on dunfell with subsequent cherry-picks.